src/main/java/reposense/system/CustomLogFormatter.java

```diff
@@ -0,0 +1,31 @@
+package reposense.system;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.logging.Handler;
+import java.util.logging.LogRecord;
+import java.util.logging.SimpleFormatter;
+
+public class CustomLogFormatter extends SimpleFormatter {
+
+    private static final DateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy hh:mm:ss");
+
+    @Override
+    public synchronized String format(LogRecord record) {
+        StringBuilder builder = new StringBuilder(1000);
```

Comment:
is it necessary to initialise the initial capacity?

Reply:
Noop